lib/aptly/account.rb

```diff
@@ -0,0 +1,16 @@
+module Aptly
+  class Account < Resource
+    def_attr :id
+    def_attr :handle
+    def_attr :apps_href, "_links", "apps", "href"
```

Comment:
where do each of these middle values (i.e. "apps", "operations", etc.) come from? Aptible has them defined somewhere?

Reply:
This is the path, in the JSON data, from the root to the thing I'm looking for. Aptible uses this approach to APIs called "hyperresource", which, once upon a time, someone [tried to make happen](https://github.com/gamache/hyperresource). Anyhow, it specifies how resources should be returned (in JSON) along with data about associated resources. So the API should be completely discoverable, blah, blah. This is part of that.

So, Aptible has its own API library for Ruby, and I'd love to use that instead of rebuilding its functionality, but it has dependencies that we need to keep out of our apps (like a forked version of the OAuth2 library).

Reply:
This sort of thing is common enough that I should probably add stuff like `def_href`.

Reply:
that sounds like a good idea